testworld/park.go

```diff
@@ -0,0 +1,248 @@
+// +build testworld
+
+package testworld
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+
+	"testing"
+
+	"time"
+
+	"github.com/centrifuge/go-centrifuge/bootstrap"
+	"github.com/centrifuge/go-centrifuge/cmd"
+	"github.com/centrifuge/go-centrifuge/config"
+	ctx "github.com/centrifuge/go-centrifuge/context"
+	"github.com/centrifuge/go-centrifuge/identity"
+	"github.com/centrifuge/go-centrifuge/node"
+	"github.com/gavv/httpexpect"
+	logging "github.com/ipfs/go-log"
+)
+
+var log = logging.Logger("host")
+
+var hostconfig = []struct {
+	name             string
+	apiPort, p2pPort int64
+}{
+	{"Alice", 8084, 38204},
+	{"Bob", 8085, 38205},
+	{"Charlie", 8086, 38206},
+}
+
+// manager is the manager of the hosts at Testworld (Robert)
+type manager struct {
```

Comment:
Could we come up with a better name?

https://stackoverflow.com/questions/1866794/naming-classes-how-to-avoid-calling-everything-a-whatevermanager

Reply:
I can make it `hostManager`